test/tests/time_integrators/actually_explicit_euler_verification/ee-1d-linear.i

```diff
@@ -0,0 +1,95 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  xmin = -1
+  xmax = 1
+  nx = 200
+  elem_type = EDGE2
+[]
+
+[Functions]
+  [./ic]
+    type = ParsedFunction
+    value = 0
+  [../]
+
+  [./forcing_fn]
+    type = ParsedFunction
+    value = x
+  [../]
+
+  [./exact_fn]
+    type = ParsedFunction
+    value = t*x
+  [../]
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+
+    [./InitialCondition]
+      type = FunctionIC
+      function = ic
+    [../]
+  [../]
+[]
+
+[Kernels]
+  [./ie]
+    type = TimeDerivative
+    variable = u
+    lumping = true
+  [../]
+
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+
+  [./ffn]
+    type = BodyForce
+    variable = u
+    function = forcing_fn
+  [../]
+[]
+
+[BCs]
+  active = 'all'
+
+  [./all]
+    type = FunctionDirichletBC
+    variable = u
+    boundary = '0 1'
+    function = exact_fn
+  [../]
+[]
+
+[Postprocessors]
+  [./l2_err]
+    type = ElementL2Error
+    variable = u
+    function = exact_fn
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+
+  start_time = 0.0
+  num_steps = 20
+  dt = 0.00005
+
+  [./TimeIntegrator]
+    type = ActuallyExplicitEuler
+  [../]
+[]
+
+[Outputs]
+  exodus = true
+  [./console]
+    type = Console
+    max_rows = 10
+  [../]
+[]
```

Comment:
How is this file testing explicit? `implicit = true` for all objects, is that flag not needed under some circumstances? It is used in your other input files...

The same comment for all other input files in this directory except `ee-2d-linear.i`.

Reply:
This TimeIntegrator does *not* require you to change `implicit` at all.  It was accidentally left in those other input files - I have removed it.

It is explicit because that's how I designed the TimeIntegrator to work.  It had to work without changing `implicit` so that we can switch back and forth between explicit and implicit time integrators.

Reply:
There's really no reason to mention it.  There are lots of other features of MOOSE _not_ turned on in those input files that we don't mention either 😄 

Almost no one currently uses the explicit capability so this isn't a thing to most people.  They don't even know that `implicit` exists.  If we talk about it / explain it then they'll get confused.  They simply don't need to know about it at all.

Reply:
If you think so, fine. But, I remember answering some emails on the mailing list related to explicit solve.

Reply:
Just today I was using `ActuallyExplicitEuler` and was very curious why I didn't see `implicit = false` in the nontime objects. @joshuahansel added documentation of this when he created `ExplicitTimeIntegrator` which I am thankful for. But I was still curious whether it was his PR (#14101) or this one that make `implicit = false` unnecessary. Now I know that it was this one. Very neat that it is no longer required so thanks for designing it this way (and I'm also thankful that it is now documented in `ExplicitTimeIntegrator.md`)

Reply:
Yeah, `implicit = false` is what cavemen use.